### PR TITLE
Execution time scipt

### PR DESCRIPTION
**Pull request by eternal-dreamer**
# Description

Program to check timing.

Fixes #342 
Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Execution time/execution.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import time
```
Comment (HarshCasper) [inline Python/Execution time/execution.py:1]: Can you explain to me the utility of this file? 

I guess we can instead have this code sample in README to better explain others on how they can use this Script.

File: Python/Execution time/Readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Execution time of task
+ Script to check how long task is running.
+
+## How to use
+From stopwatch.py import Stopwatch class in your code 
+Execution.py contains the source code to run stopwatch.py
+# How to execute execution.py
+On the terminal type,
+python execution.py
```
Comment (HarshCasper) [inline Python/Execution time/Readme.md:9]: Format the README properly.

File: Python/Execution time/Readme.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+## Execution time of task
+ Script to check how long task is running.
+
+## How to use
+From stopwatch.py import Stopwatch class in your code
```
Comment (HarshCasper) [inline Python/Execution time/Readme.md:5]: Format the File names correctly. You can mention them like `stopwatch.py`

File: Python/Execution time/Readme.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+## Execution time of task
+ Script to check how long task is running.
```
Comment (HarshCasper) [inline Python/Execution time/Readme.md:2]: Add a proper description.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Change the File Name as well.

**Comment by eternal-dreamer** [comment]
Please review it again

**Pull request merged**